.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,14 @@
+name: auto-merge
+
+on:
+  pull_request:
+
+jobs:
+  auto-merge:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ahmadnassri/action-dependabot-auto-merge@v2
+        with:
+          target: patch
+          github-token: ${{ secrets.GH_TOKEN }}
```

Comment:
`GH_TOKEN` is available across org, but I would try `DISPATCH_TOKEN` instead. For some reason `GH_TOKEN`s permission didn't quite work for actions, while dispatch one did.

Reply:
Would you like to take care of that? :)